README.md

```diff
@@ -14,9 +14,348 @@ ManageIQ plugin for the Hawkular provider.
+override_gem name, path: File.expand_path("../../manageiq-ui-classic", __dir__)
+override_gem name, path: File.expand_path("../../manageiq-schema", __dir__)
+override_gem name, path: File.expand_path("../../manageiq-api", __dir__)
+override_gem name, path: File.expand_path("../../manageiq-providers-hawkular", __dir__)
+```
+
+We also recommend some other dependencies that might come in hand for debugging:
+
+```ruby
+gem 'pry'
+gem 'pry-byebug'
+```
+
+The next step is to configure each repository:
+
+```bash
+cd ~/ManageIQ/manageiq
+./bin/setup
+cd ..
+
+cd manageiq-ui-classic
+./bin/setup
+cd ..
+
+cd manageiq-schema
+./bin/setup
+cd ..
+
+cd manageiq-api
+./bin/setup
+cd ..
+
+cd manageiq-providers-hawkular
+./bin/setup
+cd ..
+
+cd hawkular-client-ruby
+bundle install
+cd ..
+```
+
+This should be enough. To run the application, we can run this, which will run the application on background on port 3000:
+
+```bash
+bundle exec rake evm:start
+```
+
+Then, we can visit `http://localhost:3000` and sign in. If everything is working fine, you should see this:
+
+![](https://i.imgur.com/mtW7UHK.png)
+
+Sign in with the default credentials (`admin`/`smartvm`), and you should be good to go!
+
+### Updating the repositories
+
+To update the repositories, you need to go on each one and run `bin/update`. This script takes care of it:
+
+```bash
+#!/bin/bash
+
+MIQ_DIR="$HOME/manageiq"
+OLD_PWD="$(pwd)"
+
+cd $MIQ_DIR
+
+for repo in $(find $MIQ_DIR -maxdepth 1 -type d)
+do
+  echo "Updating repo $repo..."
+
+  cd $repo
+
+  if [ -d .git ]; then
+    git fetch -a
+    git pull --rebase upstream master
+    ./bin/update
+    cd -
+  fi
+done
+
+echo "Updating core..."
+
+cd $MIQ_DIR/core
+
+./bin/update
+
+cd $OLD_PWD
+
+echo "DONE."
+```
+
+## Installing Hawkular
+
+There are two ways of installing Hawkular: manually or via Docker. There are
+use cases for both, and you can mix and match for your needs, but the simplest
+way of getting it working is via Hawkinit, which downloads, configures and runs
+the hawkular server, and 0+ wildfly instances with the Hawkular agent installed
+and configured.
+
+#### Via Docker (recommended)
+
+First, we need to install [hawkinit](https://github.com/Jiri-Kremser/hawkinit),
+a CLI tool that takes care of running a Hawkular cluster for you. You can do
+that by getting it through npm, like this:
+
+```bash
+sudo npm install -g hawkinit
+```
+
+You'll also need to have Docker installed, and your group should be on the
+`docker` group:
+
+```bash
+sudo usermod -a -G docker `whoami`
+newgrp docker # to update your groups on the session
+```
+
+Then, by running `hawkinit`, you'll see something like this:
+
+![](https://i.imgur.com/cYZ3iLb.png)
+
+It will ask then some questions:
+
+* What should it use for running the services? Docker-Compose? Openshift?
+* What version of Hawkular?
+* What version of Cassandra?
+* What WildFly version?
+* Standalone or Domain mode for Wildfly?
+* How many WildFly servers?
+
```

Comment:
this is great. I think currently missing:
1. information on configuring mutable vs immutable servers - as in miq we have functionality specific to each case
2. how to start an additional server on an already running env

Reply:
I added some info here: https://github.com/cfcosta/manageiq-providers-hawkular/pull/1